.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# who must review changes to this file
+/.github/CODEOWNERS @bluejekyll @djc
```

Comment:
this file can be simplified if we use teams instead of usernames but those teams would need to be created within the hickory-dns organization

Reply:
I haven't done that before, I'd be happy to configure that.